nautobot/core/testing/api.py

```diff
@@ -692,7 +693,7 @@ class NotesURLViewTestCase(APITestCase):
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
         def test_notes_url_on_object(self):
-            if hasattr(self.model, "notes"):
+            if hasattr(self.model, "notes") and self.model is not get_user_model():
```

Comment:
`Notes.user` related name has been updated to "notes", causing `hasattr` to return `True`. This is why I had to add the extra check.

Reply:
Perhaps instead could check whether `self.model.notes` is a ForeignKey to Note model?